webpack/scenes/AnsibleCollections/Details/AnsibleCollectionDetailsInfo.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'react-bootstrap';
+import { createRows } from '../../ModuleStreams/Details/ModuleStreamDetailInfo';
+
+// using Map to preserve order
+const displayMap = new Map([
+  ['name', __('Name')],
+  ['namespace', __('Namespace')],
+  ['version', __('Version')],
+  ['checksum', __('Checksum')],
+  ['pulp_id', __('Pulp ID')],
+]);
+
+// const createRows = (details, mapping) => {
+//   const rows = [];
+//   /* eslint-disable no-restricted-syntax, react/jsx-closing-tag-location */
+//   for (const key of mapping.keys()) {
+//     rows.push(<tr key={key}>
+//       <td><b>{mapping.get(key)}</b></td>
+//       <td>{details[key]}</td>
+//     </tr>);
+//   }
+//   /* eslint-enable no-restricted-syntax, react/jsx-closing-tag-location */
+//   return rows;
+// };
```

Comment:
Same with this function.

Reply:
I think you could make a shared Detail component as well